### DISCUSSION TITLE
Consultation on Architecture

### DISCUSSION THREAD

**Discussion by CoolHandLuk** [HaishinKit/HaishinKit.swift#1619] [Q&A]
I'm seeking support with the architecture of a streaming app we're planning to build. Our goal is to create a mobile application in Flutter that integrates HaishinKit's functionalities. The app should allow live streaming with the addition of graphical overlays like a scoreboard.

I have some questions regarding HaishinKit's compatibility with Flutter, performance considerations, and best practices for integrating this library into the project. Additionally, I would like to understand if Flutter is capable of handling this setup in terms of performance, or if using native solutions might be more suitable.

I would also like to know if there’s an option for consultation or paid collaboration to discuss specific project requirements and potential technical challenges.

I would be very grateful for any assistance and recommendations on the best approach to implement HaishinKit in our app.

Thank you in advance for your response!

**Comment by shogo4405** [comment]
Hello.

When using HaishinKit, the processing itself is fundamentally native, so there should be no need to worry about performance when using it with Flutter.

However, if you need to add or modify features, development for iOS and Android, as well as Flutter plugin development, will be required, which presents some maintenance challenges.

Yes, I’m happy to answer any specific questions you may have.

For technical consultations, I provide support for implementation and development advice at $100 per hour. However,  I’m only available for around 2 hours per day on weekdays and about 16 hours over the weekend.

**Reply by CoolHandLuk** [reply]
Would it be possible to discuss this further via email or another private channel? If so, you can send me a message at appstreamgame@gmail.com. Please let me know if there's a preferred way to reach you.

**Reply by shogo4405** [reply]
I have sent an email. Please check it.
